main/src/main/res/layout/settings_ab_spinner_list_item.xml

```diff
@@ -14,21 +14,16 @@
   limitations under the License.
   -->
 
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="vertical">
-
-    <TextView
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
         android:id="@android:id/text1"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:drawablePadding="8dp"
+        android:drawableRight="@drawable/spinner_triangle"
```

Comment:
Sure, I am also going to change the commit message. Also, If it is not too much to ask please let me know if you figure it out the actual root cause of this problem or if you open a ticket. I am curious about it.